user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Alarm_table_component.md

```diff
@@ -21,12 +21,17 @@ To configure the component:
 
      In case you select history alarms, the start time and end time will also need to be specified. In case you select alarms in a sliding window, the sliding window size and refresh time will need to be configured.
 
+     > [!NOTE]
+     > From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39484-->, the sliding window size and refresh time are limited to a minimum of 1 minute and a maximum of 1 day. You can specify these parameters in minutes, hours, or days. Prior to DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6, you have the option to specify the sliding window size and refresh time in milliseconds, seconds, minutes, hours, or days.
+
    - Optionally, specify a filter for the list with the *Filters* boxes. You can either configure a filter of your own, or select *Saved filter* to use an existing shared alarm filter from the DMS. If you select to use a *Parameter index* filter, you can use “?” and “\*” as wildcards (see [Searching with wildcard characters](xref:Searching_in_DataMiner_Cube#searching-with-wildcard-characters)).
 
    - Optionally, specify the *Default sorting column*, *Default sorting order* and *Initial number of alarms to load*.
 
      > [!NOTE]
-     > If an initial number of alarms to load is specified, no grouping is applied.
+     >
+     > - If an initial number of alarms to load is specified, no grouping is applied.
+     > - From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39484-->, in case you select alarms in a sliding window, the initial number of alarms to load is limited to a minimum of 1 alarm and a maximum of 100,000 alarms. By default, this is set to 10,000.
```

Comment:
We have reverted this default of 10k since this meant no grouping by default as stated above. The default is now again empty. Also the minimum and maximum are not only applied when sliding window is active. this min and max is for the setting in general.

Reply:
```suggestion
     > - From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39484-->, the initial number of alarms to load is limited to a minimum of 1 alarm and a maximum of 100,000 alarms.
```